WebDriverAgentLib/Commands/FBElementCommands.m

```diff
@@ -629,6 +629,21 @@ + (NSArray *)routes
     isSuccessful = [element fb_selectNextOptionWithOffset:offset error:&error];
   } else if ([order isEqualToString:@"previous"]) {
     isSuccessful = [element fb_selectPreviousOptionWithOffset:offset error:&error];
+  } else if (request.arguments[@"value"]) {
+    NSString *value = request.arguments[@"value"];
+    NSString* endValue = element.wdValue;
+    if(element.wdValue == value) {
+      return FBResponseWithOK();
+    }
+    do {
+      [element fb_selectNextOptionWithOffset:offset error:&error];
```

Comment:
why do we assume the value is always the `next` element?

Reply:
Well picker wheel view is a loop right, if we reach the end then it again starts with the first value, so i keep checking the next value until the end, if i meet the same value from where i started then i will consider that as an end

Reply:
this might be suboptimal if the user known in which direction to scroll. I would rather still consider the direction hint

Reply:
Okay so i will also try reading order if user has passed and will scroll accordingly otherwise by default i will scroll next.
Is it fine?